apiv2/src/plan-marketing/infra/provider/PlanMarketingBrevo.provider.ts

```diff
@@ -34,6 +34,17 @@ export class PlanMarketingBrevoProvider implements PlanMarketingGateway {
         }
     }
 
+    async getAllCampagnes(): Promise<any[]> {
```

Comment:
```suggestion
    async getCampagnes(): Promise<any[]> {
```
si tu ne précise par `by` c'est que tu prend tout